python/test/hive/util/test_hive_ddl_extractor.py

```diff
@@ -20,7 +20,6 @@
 from dataproc_templates.hive.util.hive_ddl_extractor import \
     HiveDDLExtractorTemplate
 from datetime import datetime
-from google.cloud import storage
```

Comment:
Please revert this change. As it is not relevant to this issue.

Reply:
Done